### PR TITLE
New version: CongusBongus.CDogsSDL version 1.3.0 [FP]

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Failed
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | C-Dogs SDL |     |
| Version     | 1.3.0     |  |
| Publisher   | C-Dogs SDL Team   |       |
| ProductCode |           |     |
#### Auto-updated by [vedantmgoyal2009/winget-pkgs-automation](https://github.com/vedantmgoyal2009/vedantmgoyal2009/tree/main/winget-pkgs-automation) in workflow run [1360](<https://github.com/vedantmgoyal2009/vedantmgoyal2009/actions/runs/2253282150>)

### CODE REVIEW

File: manifests/c/CiderCollective/Cider/Nightly/1.4.5.1943/CiderCollective.Cider.Nightly.installer.yaml [hunk 1]
```diff
@@ -2,20 +2,20 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
 
 PackageIdentifier: CiderCollective.Cider.Nightly
-PackageVersion: 1.4.5-alpha.1923
+PackageVersion: 1.4.5.1943
```
Comment (Trenly) [inline manifests/c/CiderCollective/Cider/Nightly/1.4.5.1943/CiderCollective.Cider.Nightly.installer.yaml:5]:
```suggestion
PackageVersion: 1.4.5-alpha.1943
```

File: manifests/c/CiderCollective/Cider/Nightly/1.4.5.1943/CiderCollective.Cider.Nightly.locale.en-US.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
 
 PackageIdentifier: CiderCollective.Cider.Nightly
-PackageVersion: 1.4.5-alpha.1923
+PackageVersion: 1.4.5.1943
```
Comment (Trenly) [inline manifests/c/CiderCollective/Cider/Nightly/1.4.5.1943/CiderCollective.Cider.Nightly.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.4.5-alpha.1943
```

File: manifests/c/CiderCollective/Cider/Nightly/1.4.5.1943/CiderCollective.Cider.Nightly.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
 
 PackageIdentifier: CiderCollective.Cider.Nightly
-PackageVersion: 1.4.5-alpha.1923
+PackageVersion: 1.4.5.1943
```
Comment (Trenly) [inline manifests/c/CiderCollective/Cider/Nightly/1.4.5.1943/CiderCollective.Cider.Nightly.yaml:5]:
```suggestion
PackageVersion: 1.4.5-alpha.1943
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Whats up with the title?

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**